src/doorbot.cpp

```diff
@@ -43,6 +43,9 @@ void Doorbot::handleCardPresent(Card c) {
     case -1:
```

Comment:
I reckon you should fail if `result < 0` here. Any negative result from [handle_response](https://github.com/zymurgic/acnode-cl/blob/4da8919b5444c093bb147c5854754580a265558d/src/network.cpp#L35) is an error. Default success is not a great pattern here I think.

Perhaps carry on special-casing `result = -1` as access denied and any other negative result as a networking error.